### DISCUSSION TITLE
Location of nedit.rc and autoload.nm

### DISCUSSION THREAD

**Discussion by peiriannydd** [eteran/nedit-ng#177] [Q&A] [answered]
Where should these files be kept?  I put mine in $HOME/.nedit as per linux, but they are not loading when I run nedit-ng.  Is there another location?

**Comment by eteran** [comment, marked as answer]
nedit-ng's configuration directory on linux is:

`$HOME/.config/nedit-ng/`

You should be able to place the files there

Please do keep in mind two things:

1. that there is a nedit-import tool to import your old settings
2. That there is a bug in the most recent RELEASE causing a crash with some `autoload.nm` macros that has been fixed in MASTER. So using the master branch is recommended.

**Comment by peiriannydd** [comment]
I compiled MASTER yesterday, imported nedit.rc and it was put in $HOME/Library/Preferences/nedit-ng as .yaml files.  Then when I tried nedit-import autoload.nm on a simple, small autoload.nm file I got the answer
Importing is only supported for NEdit 5.6 and 5.7
Does that seem expected?

**Comment by eteran** [comment]
Ah, this is on mac, so yea, that location seems correct. As for importing `autoload.nm` just placing it in the same directory `$HOME/Library/Preferences/nedit-ng` should suffice, no need to "import it" specifically.

As for importing from previous versions of nedit, yea, I made the import tool aware of 5.6+, but didn't include any of the incremental setting's file updates that would be needed to support much older versions of nedit.

It sounds like it shouldn't be too much of an issue for you though since the main config file imported cleanly.
